src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -20,6 +20,16 @@ object AgoraEntity {
       case HttpEntity.Empty ⇒ new AgoraEntity()
     }
 
+  implicit val SeqAgoraEntityUnmarshaller =
+    Unmarshaller[Seq[AgoraEntity]](`application/json`) {
+      case HttpEntity.NonEmpty(contentType, data) => {
+        grater[AgoraEntity].fromJSONArray(data.asString)
+      }
+      case HttpEntity.Empty => {
+        Seq(new AgoraEntity())
```

Comment:
Alternately we might be able to use delegate marshalling to use the same marshall for both a single AgoraEntity and a Seq[AgoraEntity]. That might be cleaner but I've never tried it.